### DISCUSSION TITLE
I have a problem with git sync

### DISCUSSION THREAD

**Discussion by jackadam1981** [requarks/wiki#6830] [Help / Questions]
Version:2.5.300 in docker
I have correctly configured the git repository (syncDirBi), and the synchronization is normal. At least I can see the correct synchronized files in the repo directory, but I have files that cannot be found in the wiki frontend.
I've tried
purge local repository -- Force Sync
add untracked Changes
rebuild page tree
rerender all pages
But I know clearly that there is an html file that can be seen in the repo directory, but cannot be seen in the wiki frontend.
Is there anything else I can try? Reinstall if you don't want to run into problems.
By the way, most of my html files are converted from word documents using MultiDoc Converter.

**Comment by e4rthdog** [comment]
I want to use Obsidian editor and create my pages on my laptop and add it to the repo. I am adding pages but there are nowhere to be seen  in Wii.js frontend.

I can see them in the repo files in `/wiki/data/repo`
